services/game-actions.js

```diff
@@ -0,0 +1,37 @@
+import firebase from 'firebase'
+import moment from 'moment';
+
+export function createGame(game) {
+  return new Promise((resolve, reject) => {
+    firebase.database().ref(`games/${game.id}`).set({
+      //creator: user.id,
```

Comment:
This is a good idea, you should have the property `creator` so we can tell the difference between the creator and the new player.